tests/components/scorecard_chart.test.ts

```diff
@@ -66,17 +50,21 @@ async function updateScorecardChartSize(width: number, height: number) {
   await nextTick();
 }
 
-function getChartBaselineTextContent() {
-  const baseline = getChartBaselineElement()!;
-  const spans = baseline.querySelectorAll("span");
-  let text = "";
-  for (let span of spans) {
-    text += span.textContent;
-  }
-  return text;
+function getFontSize(font: string) {
+  return Number(font.match(/([0-9\.]*)px/)?.[1]);
+}
+
+function getScorecardData(chartId: UID): ScorecardChartDesign {
+  const canvas = document.createElement("canvas");
+  const figure = model.getters.getFigure(sheetId, chartId)!;
+  const runtime = model.getters.getChartRuntime(chartId) as ScorecardChartRuntime;
+  const drawer = new ScorecardChartDrawer({ figure }, canvas, runtime);
+  return drawer.computeChart();
 }
 
-describe("Scorecard charts", () => {
+let scorecardChartStyle;
+
+describe("Scorecard charts snapshots", () => {
```

Comment:
Snapshot do'nt make much sense we only have a canvas. Either snapshot the canvas instructions ore remove the test altogether